### PR TITLE
merged with upstream

### PR DESCRIPTION
**Pull request by Hermes-888**
added copyID

### CODE REVIEW

File: blossom/components/Leaderboard.php [hunk 1]
```diff
@@ -1,11 +1,27 @@
 <?php namespace Delphinium\Blossom\Components;
 
 use Cms\Classes\ComponentBase;
+use Delphinium\Roots\Roots;
+<<<<<<< HEAD
+=======
```
Comment (damarisz) [inline blossom/components/Leaderboard.php]: It seems like there are conflicts in this file. Can't merge the pull request with these merge errors.

Comment (Hermes-888) [inline blossom/components/Leaderboard.php]: Fixed leaderboard.php
Oops, I’ll get this some day…
Can you use the same request or do I need another

Comment (damarisz) [inline blossom/components/Leaderboard.php]: I think this same request is ok. I can see the file has been fixed, so let's use this one.

Comment (Hermes-888) [inline blossom/components/Leaderboard.php]: You won’t need Vanilla. Only the stuff for blossom/ assets & competencies

**Pull request merged**